lunatik.lua

```diff
@@ -52,6 +52,9 @@ driver.__runtimes = {}
 driver.__threads = {}
 
 function driver:run(script, ...)
+	if self.__runtimes [script] then
```

Comment:
please mind the code style on braces and parentheses.. e.g., `s/self.__runtimes [script]/self.__runtimes[script]/`